recipes/r-bit/meta.yaml

```diff
@@ -80,8 +80,8 @@ about:
 # Title: A class for vectors of 1-bit booleans
 # Version: 1.1-12
 # Date: 2012-01-17
-# Author: Jens Oehlschlägel <Jens.Oehlschlaegel@truecluster.com>
-# Maintainer: Jens Oehlschlägel <Jens.Oehlschlaegel@truecluster.com>
+# Author: Jens Oehlschlaegel <Jens.Oehlschlaegel@truecluster.com>
```

Comment:
No umlauts or accents any more? That is clearly a bug in conda build. I'd favor a fix in conda build instead of messing around with peoples names.